exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/transactions/CreateCounterTxIntegrationTest.java

```diff
@@ -150,26 +145,29 @@ void info() {
     AnyTransaction<CreateCounterTx> txParams = json().fromJson(info,
         new TypeToken<AnyTransaction<CreateCounterTx>>(){}.getType()
     );
-    assertThat(txParams.service_id, equalTo(QaService.ID));
-    assertThat(txParams.message_id, equalTo(CREATE_COUNTER.id()));
-    assertThat(txParams.body, equalTo(tx));
+    assertThat(txParams.service_id).isEqualTo(QaService.ID);
+    assertThat(txParams.message_id).isEqualTo(CREATE_COUNTER.id());
+    assertThat(txParams.body).isEqualTo(tx);
   }
 
-
   @Test
   void equals() {
     EqualsVerifier.forClass(CreateCounterTx.class)
         .verify();
   }
 
-  /** Creates a counter in the storage with the given name and initial value. */
-  static void createCounter(Fork view, String name, Long initialValue) {
-    HashCode nameHash = defaultHashFunction().hashString(name, UTF_8);
-    QaSchema schema = new QaSchema(view);
-    MapIndex<HashCode, Long> counters = schema.counters();
-    MapIndex<HashCode, String> counterNames = schema.counterNames();
-    counters.put(nameHash, initialValue);
-    counterNames.put(nameHash, name);
+  private TransactionMessage createCreateCounterTransaction(String counterName) {
+    CreateCounterTx createCounterTx = new CreateCounterTx(counterName);
+    RawTransaction rawTransaction = createCounterTx.toRawTransaction();
+    return toTransactionMessage(rawTransaction);
+  }
+
+  private TransactionMessage toTransactionMessage(RawTransaction rawTransaction) {
```

Comment:
This code is duplicated in several tests — could be extracted in a separate utility class.